docs/Installing.asciidoc

```diff
@@ -326,9 +326,11 @@ start` instead.
 
 == Run workers
 
-Workers are processes running virtual machines to perform the actual
-testing. They are distributed as a separate package and can be installed on
-multiple machines but still using only one WebUI.
+openQA workers are services running backends to perform the actual testing.
```

Comment:
I would rephrase the caption to "Run openQA workers" and remove the "openQA" in the paragraph. Due to the heading it should be clear enough what kind of workers the paragraph is about. This way we don't have to start the sentences weirdly with a small letter.

Reply:
ok, fine. I have followed that advice but now I changed the reference to the package as I think it can be confusing to read that "they can be installed" when actually the (single) package is installed. Please check again the updated text.